front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -98,7 +96,17 @@ const IdeasShowPage = () => {
     ).length > 0;
   const showCTABar =
     isIdeaInCurrentPhase && phase?.attributes.participation_method === 'voting';
-  const showCTABarAtTopOfPage = !isSmallerThanTablet && showCTABar;
+
+  const getPaddingTopIdeaContainer = () => {
```

Comment:
Realization after adding the comments below: it would actually be much clearer if we added margin-bottom to the elements that are causing us to add padding-top to the container instead. That way, we also wouldn't have to ensure the conditions here and in the return statement are kept in sync. I feel like this will cause bugs in the future.

Reply:
Padding still works then I think? Would have to look deeper into this but trying to keep my focus on platform week.

Reply:
I mean: padding or margin, whatever works. :) I haven't tried anything.

Reply:
Can take a quick look first thing tomorrow. Let me know when I shouldn't.

Reply:
Sure that's fine! :) I think I might be misunderstanding your suggestion.

Reply:
The TL;DR is: we have too many conditions in this component, spread out over too many places.

With this PR, some extra conditions are going in `getPaddingTopIdeaContainer`, so each time we're making changes in the return statement, we need to think of checking `getPaddingTopIdeaContainer`. I think there's a good chance this will lead to more bugs in the future.

What I'm suggesting, as a first step, is to bring the conditions and styles closer to the places to which they apply.  Here's a PR that demonstrates what I mean: https://github.com/CitizenLabDotCo/citizenlab/pull/9835/files.

Like I said, this is a first step, because at this point, with all these cases to deal with, we should look into splitting up the return statement between a version when isSmallerThanTablet is true and when it's not. This will further simplify the code and make it easier to reason about and not introduce bugs.